.flake8.cython

```diff
@@ -1,4 +1,4 @@
 [flake8]
 filename = *.pyx, *.pxd, *.pxi
 exclude = .git, .eggs, *.py
-ignore = W503,W504,E225,E226,E227,E275,E402,E999
+ignore = W503,W504,E211,E225,E226,E227,E275,E402,E999
```

Comment:
E211 added to avoid `Whitespace before '('` error:

```
ctypedef nvrtcResult (*F_nvrtcVersion)(int *major, int *minor) nogil
                    ^
```